recipes/ffmpeg/all/conanfile.py

```diff
@@ -270,6 +275,8 @@ def requirements(self):
             self.requires("openjpeg/2.5.0")
         if self.options.with_openh264:
             self.requires("openh264/2.3.1")
+        if self.options.get_safe("with_openvino"):
+            self.requires("openvino/2023.2.0")
```

Comment:
From https://github.com/FFmpeg/FFmpeg/commit/87cb24a1ca4a76e5a5a9969e3058a94d952e1b37:
> 1. install required OpenCL drivers, see: https://github.com/intel/compute-runtime/releases/tag/19.41.14441
> 2. build OpenVINO c lib with GPU enabled: use cmake config with: -DENABLE_CLDNN=ON
> 3. then make, and include the OpenVINO c lib in environment variables

Do you have to force ENABLE_CLDNN here?

Reply:
it's enabled by default in OpenVINO